web/docs/auth/auth-hooks.md

```diff
@@ -828,9 +888,31 @@ Wasp passes the `oauth` object to the `onAfterSignup` and `onAfterLogin` hooks o
 
 It has the following fields:
 
-- `accessToken: string`
+- `providerName: string`
+
+  The OAuth provider name (e.g. `'google'`, `'github'`) that the user authenticated with.
```

Comment:
```suggestion
  The name of the OAuth provider the user authenticated with (e.g. `'google'`, `'github'`).
```
This formulation is less confusing IMO. The original soulds like the user authenticated with an "OAuth provider name."
Plus, the examples are in the middle of the sentence, breaking the flow.